Rakefile

```
@@ -21,7 +21,13 @@ require 'aufgaben/release'
 Aufgaben::Release.new
 
 require 'aufgaben/bump/ruby'
-Aufgaben::Bump::Ruby.new
+Aufgaben::Bump::Ruby.new do |t|
+  t.files = %w[
+    .ruby-version
+    base/Dockerfile.prepare.erb
+  ] + Dir.glob("**/Dockerfile")
```

[suggestion] Is it better to rely on `rake dockerfile:generate` for the update of the Dockerfiles?

```suggestion
  ]
```

---

Without changing these files, can we merge the change only with `bundle exec rake bump:ruby'[VERSION]'`? I rarely run `docker:build` task on my local machine if I want to just update the Ruby version.

---

I think it better to have a single responsibility for one task, because:

- We have ran `bundle exec rake dockerfile:generate dockerfile:verify` on CI already.
- We should run `bundle exec rake bump:ruby'[VERSION]' dockerfile:generate` if we want to update the Ruby version.

---

Fair enough.